src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -31,6 +34,41 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the endpoint supports glob expressions
+     * @param endpointValue the endpoint value
+     * @return <code>true</code> if the endpoint supports glob expressions , <code>false </code> otherwise
+     */
+    public static boolean isGlobExpressionsSupported(Endpoint endpointValue) {
+        return (Endpoint.JUNIT.equals(endpointValue)
+                || Endpoint.NUNIT.equals(endpointValue)
+                || Endpoint.TESTNG.equals(endpointValue)
+                || Endpoint.ROBOT.equals(endpointValue)
+                || Endpoint.XUNIT.equals(endpointValue)
+                || Endpoint.JUNIT_MULTIPART.equals(endpointValue)
+                || Endpoint.ROBOT_MULTIPART.equals(endpointValue)
+                || Endpoint.TESTNG_MULTIPART.equals(endpointValue)
+                || Endpoint.NUNIT_MULTIPART.equals(endpointValue));
+    }
+
+    /**
+     * Utility method to get the generic endpoint from the corresponding multipart endpoint.
```

Comment:
This is not very clear what you mean by this...
I suggest you to add 1 or 2 examples to this JavaDoc